test/test_nn.py

```diff
@@ -805,6 +805,15 @@ def test_no_grad(self):
             self.assertFalse(output2.requires_grad)
             self.assertRaises(RuntimeError, lambda: output2.backward(torch.ones(1, 5, 10, 10)))
 
+    def test_invalid_conv1d(self):
```

Comment:
To close #17247, could you please add all the reported failed use cases to the test?

Reply:
We already have other test cases e.g. invalid_conv2d and 3d which does check for current error